packages/builder/src/balance/BalanceBuilder.ts

```diff
@@ -164,7 +164,10 @@ export async function calculateSystemAccountBalance(
   const balance = response.data as PalletBalancesAccountData &
     PalletBalancesAccountDataOld;
 
+  const free = BigInt(balance.free.toString());
```

Comment:
Can you please add some unit tests for this function?

Reply:
In `BalanceBuilder.test.ts` we have tests for `system.account`, which basically calls only this function. Do you want to write unit tests separately for the specific function?

Reply:
To be honest for this specific func I would like to have separate even if you generate a test using AI.

Reply:
added